src/components/Splash.jsx

```diff
@@ -0,0 +1,49 @@
+import { useEffect } from 'react';
+import { BiChevronRightCircle } from 'react-icons/bi';
+import houseImage from '../assets/house.png';
+
+const Splash = () => {
+  useEffect(() => {
+    const menu = document.querySelector('#menu');
+    menu.style.display = 'none';
+    const nav = document.querySelector('nav');
+    nav.style.display = 'none';
+
+    const mediaQuery1 = window.matchMedia('(min-width: 1340px)');
+
+    const handleMediaQueryChange = (event) => {
+      if (event.matches) {
+        const nav = document.querySelector('nav');
+        nav.style.display = 'none';
+      }
+    };
+
+    mediaQuery1.addEventListener('change', handleMediaQueryChange);
+
+    return () => {
+      mediaQuery1.removeEventListener('change', handleMediaQueryChange);
+    };
+  }, []);
+
+  return (
+    <div className="w-screen h-screen bg-center bg-no-repeat" style={{ backgroundImage: `url(${houseImage})` }}>
```

Comment:
I'll suggest you make the buttons a little bigger since its not visible enough with the background image.

Reply:
You're right, teammate, I will make those buttons bigger to increase accessibility. 🔧